atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -207,7 +205,7 @@ private boolean isHostHealthy(InetSocketAddress host) {
         try {
             CassandraClientPoolingContainer testingContainer = new CassandraClientPoolingContainer(host, config);
             testingContainer.runWithPooledResource(describeRing);
-            testingContainer.runWithPooledResource(CassandraVerifier.healthCheck);
+            testingContainer.runWithPooledResource(validatePartitioner);
```

Comment:
Is CassandraVerifier.healthCheck still used (actually seems to describe_version)?